NDTensors/src/lib/TypeParameterAccessors/src/parameters.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    parameters(type::DataType)
+
+Gets all the type parameters of the DataType `type`.
+"""
+parameters(type::DataType) = Tuple(type.parameters)
```

Comment:
I wonder if we need to convert to `Tuple` here.

Reply:
In fact it seems like we may be able to just remove this function in favor of other functions, the primary use case was `set_parameter` but the current implementation of that doesn't use this (which I think is a good thing).

Reply:
I believe we could remove this function, I am working on that now.